.github/workflows/updateZoweVersionsAndCreatePR.yml

```diff
@@ -0,0 +1,96 @@
+name: Update Zowe Component Versions and Create PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release Version (x.x.x)'
+        required: true
+      zowe_version:
+        description: "Zowe Version Tag"
+        default: zowe-v2-lts
+        required: true
+        type: choice
+        options:
+        - zowe-v1-lts
+        - zowe-v2-lts
+
+jobs:
+  update_versions_and_create_pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set Default Branch Environment Variable
+        run: |
+          if [[ "${{ github.event.inputs.zowe_version }}" == "zowe-v1-lts" ]]; then
+            echo "defaultBranch=v1.x/staging" >> $GITHUB_ENV
+          else
+            echo "defaultBranch=v2.x/staging" >> $GITHUB_ENV
+          fi
+
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ env.defaultBranch }}
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update -y
+          sudo apt install gh

```

Comment:
this step ("Install Dependencies") can be removed since GH CLI is already provided with `ubuntu-latest`